IGI.Lab3/CarPark.WebUI/Views/Car/Delete.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@model CarPark.WebUI.ViewModels.CarViewModel
+
+@{
+    ViewData["Title"] = "Delete car";
+}
+
+<h1>Delete</h1>
+
+<h3>Are you sure you want to delete this?</h3>
+<div>
+    <h4>Car</h4>
+    <hr />
+    <dl class="row">
+        <dt class="col-sm-2" asp-display-for="Id">
+        </dt>
+        <dd class="col-sm-10" asp-display-name-for="Id">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="Color">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="Color">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="Rented">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="Rented">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="CarRegistrationNumber">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="CarRegistrationNumber">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="Price">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="Price">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="Class">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="Class">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="Model">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="Model">
+        </dd>
+        <dt class="col-sm-2" asp-display-name-for="CarMake">
+        </dt>
+        <dd class="col-sm-10" asp-display-for="CarMake">
```

Comment:
For the near future (PR 4), it can be simplified.
For example:
```html
<dt class="col-sm-2" asp-display-name-for="Model" />
<dd class="col-sm-10" asp-display-for="Model" />
```